### ISSUE TITLE
Markdown-specific softWrap setting ignored when clicking in a file

### ISSUE THREAD

**Issue by alex-r-bigelow** [burodepeper/language-markdown#256]
To reproduce, edit `.config` to look like:
```
"*"
  editor:
    softWrap: false

...

".md.text"
  editor:
    softWrap: true
```

Opening a markdown file will initially be soft-wrapped, but clicking in the file seems to apply the Atom-wide setting.

![markdown](https://user-images.githubusercontent.com/1215873/77115513-b994f700-69eb-11ea-8fba-c169e3bb8a98.gif)

I think there are a few other scenarios that trigger this switch; I'm having to constantly re-enable soft wrap manually whenever I'm in a markdown file. Note that custom softWrap settings work for other formats like LaTeX.

**Comment by burodepeper** [comment]
Have you researched or cross posted this to the repo that handles the soft wrapping? Might be the general Atom repo, not sure. I'm not aware that this package is actively interfering with wrapping.

**Comment by alex-r-bigelow** [comment]
I remember seeing some odd bugs filed under the core repository that are probably relevant, but as this seems to only be affecting markdown files, I figured the problem lived here? I saw a couple references in this repo to soft wrapping settings, so my immediate guess was that there were probably some breaking changes somewhere upstream that hadn't yet been accounted for? I'm still really new to the internals of Atom, so I really don't have a good sense where this would be coming from.

**Comment by burodepeper** [comment]
Thanks for looking through earlier issues!

Try filing it in the core repository first. If that doesn't resolve anything, I could have a quick look as well. I remember having some issues with soft wrap in the past, but I've not actively been using Atom for a while now.

Additionally, what other markdown related packages do you have installed?

Does the problem persist if you switch to the core language-gfm package and disable language-markdown?

**Comment by alex-r-bigelow** [comment]
Yes, the problem does go away if I switch and disable `language-markdown`. I don't think I have any other markdown-related packages other than `markdown-preview` (and disabling it doesn't seem to change anything). Here are the contents of `~/.atom/packages` if it's helpful:

```
atom-beautify
bombe
busy-signal
color-picker
colorbrewer-8-dark2-syntax
editor-settings
intentions
language-docker
language-glsl
language-latex
language-markdown
linter
linter-docker
linter-eslint
linter-pylint
linter-ui-default
multi-cursor
wordcount
```

Also, in case it's relevant, I'm running Atom v1.45.0 on Windows 10, and here's my full `config.cson`:

<details><summary></summary><pre>
"*":
  "atom-beautify":
    general:
      _analyticsUserId: "309581cc-293d-4a9e-b1d3-40a7487115c9"
  "autocomplete-plus":
    confirmCompletion: "tab always, enter when suggestion explicitly selected"
  core:
    disabledPackages: [
      "linter-pylint"
      "language-gfm"
    ]
    restorePreviousWindowsOnStart: "no"
    telemetryConsent: "limited"
    themes: [
      "one-dark-ui"
      "colorbrewer-8-dark2-syntax"
    ]
    uriHandlerRegistration: "always"
  editor:
    fontSize: 11
    softWrap: false
  "exception-reporting":
    userId: "703574ec-d6a8-4838-9929-bb29448dbee0"
  "line-ending-selector":
    defaultLineEnding: "LF"
  "linter-eslint":
    disabling:
      disableWhenNoEslintConfig: false
    global:
      eslintrcPath: "C:\\Users\\Alex\\AppData\\Local\\Packages\\CanonicalGroupLimited.UbuntuonWindows_79rhkp1fndgsc\\LocalState\\rootfs\\home\\home\\.eslintrc"
      globalNodePath: "C:\\Users\\Alex\\AppData\\Roaming\\npm"
      useGlobalEslint: true
  "linter-pylint":
    executablePath: "C:\\Users\\Alex\\Anaconda3\\Scripts\\pylint"
    pythonPath: "%p\\env\\bin\\pylint"
    rcFile: "C:\\Users\\Alex\\.pylintrc"
  "linter-ui-default":
    showPanel: true
  "markdown-preview":
    useGitHubStyle: true
  "spell-check":
    addKnownWords: true
    grammars: [
      "source.asciidoc"
      "source.gfm"
      "text.git-commit"
      "text.plain"
      "text.plain.null-grammar"
      "source.rst"
      "text.restructuredtext"
      "text.tex.latex"
      "text.md"
    ]
    knownWords: [
      "jupyter"
    ]
  "tree-view": {}
  welcome:
    showOnStartup: false
".latex.tex.text":
  editor:
    softWrap: true
    softWrapAtPreferredLineLength: true
".md.text":
  editor:
    softWrap: true
    softWrapAtPreferredLineLength: true

</pre></details>

**Comment by burodepeper** [comment]
I see nothing out of the ordinary in your packages or config. I tried recreating your issue, but I can't get it to fail on my machine... Weird... Try posting it to the core package. I don't think I can be of much help to you.
